halo/cmd/ready.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+
+	"github.com/spf13/cobra"
+)
+
+type readyConfig struct {
+	MonitoringAddr string
+}
+
+func defaultReadyConfig() readyConfig {
+	return readyConfig{
+		MonitoringAddr: "http://localhost:26660",
+	}
+}
+
+func newReadyCmd() *cobra.Command {
+	cfg := defaultReadyConfig()
+
+	cmd := &cobra.Command{
+		Use:   "ready",
+		Short: "Query node for readiness",
```

Comment:
`Query remote node for readiness`

Reply:
But is it "remote" if run the command on my halo node that queries localhost?

Reply:
Discussed in Slack